website/routes.py

```diff
@@ -257,6 +257,17 @@ def ember_app(path=None):
                         preprint = Preprint.load(guid_str)
                         if preprint and preprint._id != guid_str:
                             return redirect(f"{settings.DOMAIN}preprints/{path_values[0]}/{preprint._id}", code=302)
+                        if preprint:
+                            from framework.auth import Auth
+                            auth = Auth.from_kwargs(request.args.to_dict(), {})
+                            if not preprint.can_view(auth):
+                                if getattr(preprint, 'is_pending_moderation', True):
```

Comment:
is this necessary? Can't it be simplified to 

```suggestion
                                if preprint.is_pending_moderation:
```